src/frontend/admin/src/components/testing/presentational/filters/SearchFilters/SearchFilters.spec.e2e.tsx

```diff
@@ -1,14 +1,17 @@
 import { expect, test } from "@playwright/experimental-ct-react";
 import { Page } from "@playwright/test";
+import mockRouter from "next-router-mock";
+import { HooksConfig } from "../../../../../../playwright";
 import { SearchFiltersWrapperTest } from "@/components/testing/presentational/filters/SearchFilters/SearchFiltersWrapperTest";
+import { PlaywrightCustomRouter } from "@/components/testing/PlaywrightCustomRouter";
 
 const selectInputs = async (page: Page) => {
   await page.getByLabel("Yes").check();
   await page.getByLabel("Language").click();
   await page.getByRole("option", { name: "French" }).click();
   await page.getByLabel("User").click();
   await page.getByTestId("custom-modal").getByLabel("User").fill("jo");
-  await page.getByRole("option", { name: "John Doe" }).click();
+  await page.getByRole("option", { name: "JohnDoe" }).click();
```

Comment:
Why have you changed John Doe ? :p

Reply:
To not add special characters for spaces